kieserver/image.yaml

```diff
@@ -142,6 +142,12 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
+    example: "true"
+    description: "Enable the KIE Server Jbpm clustering for processes fail-over. If the number of replicas is less than two, one additional replica will be started."
+  - name: "KIE_SERVER_JBPM_CLUSTER_TRANSPORT_LOCK_TIMEOUT"
+    value: "60000"
+    description: "Jbpm Cache transport lock timeout, value in milliseconds. The default value is 6000 ms."
```

Comment:
Does 60000 value is 6000ms?
I see that incase user doesn't define anything 60000 value is taken: https://github.com/jboss-container-images/jboss-kie-modules/pull/434/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cR737

Reply:
60000 is taken from the configuration here https://issues.redhat.com/browse/BAPL-1741

Reply:
edit the description to match the default value.

Reply:
and for example, try to use a different value than the default.

Reply:
Done